### PR TITLE
fix: Performance testing script updates

### PR DESCRIPTION
**Pull request by saraburns1**
1. Add check to compare count of slices to rowcount from clickhouse to ensure data for all slices has been returned
2. Add 'strip' to remove whitespace that caused memory=none issues
3. Simplify ch query

### CODE REVIEW

**Review by Ian2012 — APPROVED** [review]

File: tutoraspects/templates/aspects/apps/superset/pythonpath/performance_metrics.py [hunk 1]
```diff
@@ -228,6 +229,17 @@ def get_query_log_from_clickhouse(report, query_contexts, print_sql, fail_on_err
 
     ch_chart_result = measure_chart(slice, query_context, fail_on_error)
 
+    # Run CH query until results for all slices are returned
+    ch_count = 6
+    while ch_count > 0:
+        if ch_chart_result["queries"][0]["rowcount"] < chart_count:
+            logger.info("Waiting for clickhouse log...")
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/apps/superset/pythonpath/performance_metrics.py:236]: Can you dump how many we're still waiting for here as well?

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/query_log.yaml [hunk 2]
```diff
@@ -118,7 +118,7 @@ sql: |
       query,
       http_user_agent
   FROM system.query_log
-  WHERE (has(databases, '{{ASPECTS_XAPI_DATABASE}}') OR has(databases, '{{ASPECTS_EVENT_SINK_DATABASE}}') OR has(databases, '{{DBT_PROFILE_TARGET_DATABASE}}')) AND (http_user_agent LIKE 'aspects-%') AND (type = 'QueryFinish')
+  WHERE databases <> ['system'] AND (http_user_agent LIKE 'aspects-%') AND (type = 'QueryFinish')
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/query_log.yaml:121]: I'm not sure how much it matters, but there are some installs that have other things in ClickHouse aside from Aspects and this change might make things slower there.

**Review by bmtcril — APPROVED** [review]
Just a couple of nits 👍

**Comment by bmtcril** [comment]
Any idea why the SQL is empty? 


```
40. learner-groups - Enrollees by Enrollment Track f8beeb8c-b35c-47fc-8370-96b493f9c1b8
Superset time: 0.13573 (s).
Query duration: 0.012 (s).
Result rows: 3
Memory Usage (MB): 4.420504570007324
Row count (superset) 3
Filters: []
SQL:


```

**Comment by saraburns1** [comment]
you have to specify the option to print the sql

```"--print_sql", is_flag=True, default=False, help="Whether to print the SQL run."```

**Pull request merged**